public/src/pages/views/CourseView.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from "react";
 import { Course } from "../../../proto/ag_pb";
 import { DynamicTable, Search } from "../../components";
-import { searchForCourses } from "../../componentHelper";
+import { searchForCourses } from '../../componentHelper';
```

Comment:
What's the difference between single and doble quotes? Should probably be consistent with one type of quote, but if we want to move to single quote for some specific reason, then this is fine...

Reply:
Well, I guess it is just manual and automatic imports done in different styles. I'll fix it.

Reply:
Done.